### ISSUE TITLE
Example notebook JSON is broken

### ISSUE THREAD

**Issue by matthewfeickert** [scikit-hep/mplhep#227]
It seems that PR #222 broke the JSON in the Example notebook. If you try to launch it you're hit with a 

```
File Load Error for Examples.ipynb

Unreadable Notebook: /home/jovyan/mplhep/examples/Examples.ipynb NotJSONError('Notebook does not appear to be JSON: \'{\\n "cells": [\\n {\\n ...')
```

![broken_notebook](https://user-images.githubusercontent.com/5142394/98841938-6c4c1680-240e-11eb-9757-089e0e1cbf90.png)

but if you checkout the previous commit that touched it

```
$ git log --oneline -- examples/Examples.ipynb
1970b99 feat: add LHCb style 2 (#222)
3fb4f35 chore: simplify 1 vs multi hist handling
358b05f fix: #177 (#184)
a2aab86 feat: improve label names, update tests
$ git checkout 3fb4f35de18e6b00c83cb1bd7d21dc3810aff815 -- examples/Examples.ipynb
```

then you're able to open it fine.

**Comment by matthewfeickert** [comment]
(totally unrelated, but those are some good looking plots in PR #222 :slightly_smiling_face:)

**Comment by matthewfeickert** [comment]
The good news is that it just seems that the problem is that single quotes were used where double quotes should have been. 

https://github.com/scikit-hep/mplhep/pull/222/files#diff-fda14a2cc26373fd77619a496b36ea2ac734cd2ee0aa0bc694b40c2f810eb32aR84

Not sure if it would be easier to go in and manually fix those or if it would be easier/faster to just rollback the notebook and then have Jonas redo his changes while they're still fresh in his mind. @mayou36 thoughts?

**Comment by mayou36** [comment]
No idea why it broke. In fact, we had this odd diff, which I reverted.

Anyway, it runs for me. I've just rerun it and committed again

**Comment by mayou36** [comment]
Should be fixed now, can you confirm @matthewfeickert ?

**Comment by andrzejnovak** [comment]
I disabled the papermill tests some time back due to the upstream bug, but should probably check if they would work again.

**Comment by matthewfeickert** [comment]
> Should be fixed now, can you confirm @matthewfeickert ?

Fixed in PR #228 :+1: 

> I disabled the papermill tests some time back due to the upstream bug, but should probably check if they would work again.

Yeah, probably worth doing so. The `v2.x` API of `papermill` seems fine for `pyhf`.
